### PR TITLE
Refactor/psd 2562 remove use new search role

### PR DESCRIPTION
**Pull request by meeravnagesh**
changes to remove the use_new_search role from code and roll out to all users

### CODE REVIEW

File: app/controllers/investigations_controller.rb [hunk 1]
```diff
@@ -10,16 +10,16 @@ class InvestigationsController < ApplicationController
...
     # Find the most recent incomplete bulk products upload for the current user, if any
-    @incomplete_bulk_products_upload = BulkProductsUpload.where(user: current_user, submitted_at: nil).order(updated_at: :desc).first
+    # @incomplete_bulk_products_upload = BulkProductsUpload.where(user: current_user, submitted_at: nil).order(updated_at: :desc).first
 
-    @pagy, @answer  = pagy_searchkick(opensearch_for_investigations(20, paginate: true))
-    @count          = count_to_display
-    @investigations = InvestigationDecorator
-                        .decorate_collection(@answer.includes([{ owner_user: :organisation, owner_team: :organisation }, :products]))
-    @page_name = "all_cases"
+    # @pagy, @answer  = pagy_searchkick(opensearch_for_investigations(20, paginate: true))
```
Comment (alan-at-work) [inline app/controllers/investigations_controller.rb:18]: Remove these comments, code is no longer needed

File: spec/features/cases_spec.rb [hunk 2]
```diff
@@ -39,7 +39,7 @@
 
     expect(page).to have_css(".govuk-pagination__link", text: "1")
 
-    expect(page).to have_link("Next", href: /#{Regexp.escape(all_cases_investigations_path(pagination_link_params))}/)
+    # expect(page).to have_link("Next", href: /#{Regexp.escape(all_cases_investigations_path(pagination_link_params))}/)
```
Comment (alan-at-work) [inline spec/features/cases_spec.rb:42]: Just remove the code if it's not being used. We can always refer to the repo history if we need to :)

**Review by alan-at-work — COMMENTED** [review]
Just need to remove some commented out code, if we don't need it let's ditch it. 

If we need it back we can refer to the repo history.

File: spec/features/cases_spec.rb [hunk 3]
```diff
@@ -51,7 +51,7 @@
     expect(page.find("select#sort_by option", text: "Relevance")).to be_selected
     expect(page).to have_css("form#cases-search-form dl.opss-dl-select dd", text: "Active: Relevance")
 
-    expect(page).to have_current_path(investigations_search_path, ignore_query: true)
+    # expect(page).to have_current_path(investigations_search_path, ignore_query: true)
```
Comment (tahir-khalid) [inline spec/features/cases_spec.rb:54]: Same as Alan's comment remove the commented out code if not needed.

File: app/controllers/investigations_controller.rb [hunk 4]
```diff
@@ -10,16 +10,8 @@ class InvestigationsController < ApplicationController
...
-    # Find the most recent incomplete bulk products upload for the current user, if any
-    @incomplete_bulk_products_upload = BulkProductsUpload.where(user: current_user, submitted_at: nil).order(updated_at: :desc).first
-
-    @pagy, @answer  = pagy_searchkick(opensearch_for_investigations(20, paginate: true))
-    @count          = count_to_display
-    @investigations = InvestigationDecorator
-                        .decorate_collection(@answer.includes([{ owner_user: :organisation, owner_team: :organisation }, :products]))
-    @page_name = "all_cases"
+    redirect_to notifications_path
+    # removed the method code as part of use_new_serach roll out
```
Comment (alan-at-work) [inline app/controllers/investigations_controller.rb]: Not needed

File: package.json [hunk 5]
```diff
@@ -9,9 +9,9 @@
   "dependencies": {
     "@hotwired/stimulus": "^3.2.2",
     "accessible-autocomplete": "https://github.com/OfficeForProductSafetyAndStandards/accessible-autocomplete-multi/tarball/main",
-    "esbuild": "^0.22.0",
+    "esbuild": "^0.21.5",
```
Comment (alan-at-work) [inline package.json]: downgrade?

File: package.json [hunk 6]
```diff
@@ -9,9 +9,9 @@
...
     "govuk-frontend": "5.4.0",
-    "postcss": "^8.4.39",
+    "postcss": "^8.4.38",
```
Comment (alan-at-work) [inline package.json]: downgrade?

**Review by alan-at-work — APPROVED** [review]

**Pull request closed without merging**